app/views/topical_events/show.html.erb

```diff
@@ -161,49 +161,6 @@
                 </div>
               <% end %>
             </div>
-
-            <% if @topical_event.slug == 'first-world-war-centenary' %>
```

Comment:
Could the commit message here be made slightly clearer, as someone coming back later might not realise what 'snowflake pages' are or what they are related to.